content/marketing-assets/page/new_tab_learn_more/style.css

```
@@ -0,0 +1,165 @@
+.mkt-main {
+  padding: 50px 100px 100px;
+  background-color: #f4f4f4;
+}
+
+section.mkt-wrapper {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-column-gap: 1.5rem;
+  align-items: center;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+section.mkt-wrapper .tiny {
+  grid-row: 1;
+  grid-column: 1 / span 6;
+  font-size: .75rem;
+  margin-top: 0;
+  margin-bottom: .25rem;
+  text-transform: uppercase;
+  font-family: "Graphik Web", "Helvetica Neue", Helvetica, Arial, Sans-Serif;
+  font-size: .875rem;
+  font-weight: 400;
+  letter-spacing: 0.014em;
+  text-align: left;
+  margin-left: 0;
+  margin-right: 0;
+}
+
+section.mkt-wrapper h2 {
+  font-size: 2.25rem;
+  grid-row: 2;
+  grid-column: 1 / span 6;
+  font-weight: 500;
+  font-family: "Doyle", Garamond, Times, Serif;
+  line-height: 120%;
+  margin-top: 0;
+  margin-bottom: .25rem;
+}
+
+section.mkt-wrapper p {
+  font-size: 1.3rem;
+  grid-row: 3;
+  grid-column: 1 / span 6;
+  font-family: "Graphik Web", "Helvetica Neue", Helvetica, Arial, Sans-Serif;
+  line-height: 150%;
+}
+
+.mkt-button-wrapper {
+  grid-row: 4;
+  grid-column: 1 / span 6;
+}
+
+.new-tab-landing .mkt-cta {
+  padding-left: 1.65rem;
+  padding-right: 2.4rem;
+  font-size: 1.1875rem;
+  padding-top: .75rem;
+  padding-bottom: .75rem;
+  background-color: #ef4056;
+  color: #ffffff;
+  display: inline-block;
+  position: relative;
+  font-family: Graphik Web, Helvetica Neue, Helvetica, Arial, Sans-Serif;
```

multi-word font names should be quoted:
```suggestion
  font-family: "Graphik Web", "Helvetica Neue", Helvetica, Arial, Sans-Serif;
```